Octokit/Models/Request/LabelUpdate.cs

```diff
@@ -18,13 +18,15 @@ public class LabelUpdate
         /// </summary>
         /// <param name="name">The name of the label.</param>
         /// <param name="color">The color of the label.</param>
-        public LabelUpdate(string name, string color)
+        /// <param name="description">The description of the label.</param>
+        public LabelUpdate(string name, string color, string description = null)
```

Comment:
To be consistent with the rest of the library, the request model constructor should only take required parameters. Optional parameters can be set via object initializer pattern